packages/stylelint-config/README.md

```diff
@@ -19,27 +19,113 @@ npm install --save-dev @lmc-eu/stylelint-config stylelint-config-prettier
 > We assume you are using Prettier. That's why we also recommend adding
 > `stylelint-config-prettier` above.
 
-## Configurations
+New configuration extends community maintained config [stylelint-config-standard-scss](https://github.com/stylelint-scss/stylelint-config-standard-scss).
+
+This config extends another community maintained configs:
+
+- [stylelint-config-recommended](https://github.com/stylelint/stylelint-config-recommended)
+- [stylelint-config-recommended-scss](https://github.com/stylelint-scss/stylelint-config-recommended-scss)
+- [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard)
```

Comment:
This made me confused—I was thinking it's about _this_ config, not about `stylelint-config-standard-scss`.

Could you maybe improve it so the structure is clearer? Community configs have it like this:

<img width="783" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/191070782-0ae9917c-2df3-4294-b079-85f8a4f29eaf.png">

Reply:
Good point, it's been re-texted.